### ISSUE TITLE
[Bug] incorrect order of instances if multiframe

### ISSUE THREAD

**Issue by olivert1969** [OHIF/Viewers#3895]
### Describe the Bug

Order of instances appears to be random when each instance is multiframe

### Steps to Reproduce

Open series with multiframe instances, e.g. typical US

### The current behavior

Order of instances appears to be random and is not sorted by InstanceNumber.

### The expected behavior

Order of instances is sorted by InstanceNumber

### OS

Windows 10

### Node version

18.12

### Browser

Chrome 120

**Comment by olivert1969** [comment]
Proposed solution 

update extensions/default/src/getSopClassHandlerModule.js

add 
InstanceNumber: instance.InstanceNumber
To 
(24) imageSet.setAttributes

and

after 
displaySets.push(displaySet);
Inside (117) if (isMultiFrame(instance)) 

add 
displaySets.sort((a, b) => (a.InstanceNumber > b.InstanceNumber ? 1 : -1));

**Comment by salimkanoun** [comment]
Agreed there is a bug for stack viewport, instances should be ordered by instance number
